README.md

```diff
@@ -2,8 +2,122 @@
 
 The EA Forum is a synced fork of [LessWrong](https://github.com/LessWrong2/Lesswrong2).
 
-## Contributing
+## Technologies
 
-We have not prioritized writing a guide to the tech. You can see the instructions on the LessWrong repo, but any given sentence is more likely than not to be out of date and misleading. If you are at least moderately interested in contributing, message [JP](https://forum.effectivealtruism.org/users/jpaddison). I won't mind the inquiry, and can help you out and in the process might update this readme.
+The EA Forum is built on top of a number major open-source libraries.
 
-We do not use github for issues. Please [contact us](https://forum.effectivealtruism.org/contact) with bug reports.
+1. [Vulcan](http://vulcanjs.org/) is a framework for designing social applications like forums and news aggregators. We started out using it as a library in the usual way, then forked its codebase and diverged considerably.
```

Comment:
Is this still a relevant technology to be highlighting straight off the bat? Given _how_ much we've diverged, it might be more confusing than illuminating? Not confident here.

Reply:
I've updated it to say
> Read their docs to understand the patterns we use.